webapp/templates/partials/scoreboard_table.html.twig

```diff
@@ -358,6 +399,16 @@
                 {{ cMax }} 96%);
         }
     {% endfor %}
+
```

Comment:
Why not this to the [general CSS file](https://github.com/DOMjudge/domjudge/blob/master/webapp/public/style_domjudge.css)?

Reply:
Sound's great.